mrp_multi_level/data/mrp_area_data.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" ?>
```

Comment:
why? I don't like this change

Reply:
It's deprecated in v17 and gives a warning on the log

Reply:
Which warning? I see a lot of files with this in odoo, see for example https://github.com/odoo/odoo/blob/17.0/odoo/addons/base/data/ir_cron_data.xml#L1

Reply:
```
2024-03-15 11:37:47,883 13893 WARNING odoo_17 py.warnings: /home/jasmins/workspace/odoo_17/odoo/odoo/addons/base/models/ir_module.py:178: DeprecationWarning: XML declarations in HTML module descriptions are deprecated since Odoo 17, account_payment_order can just have a UTF8 description with not need for a declaration.
  File "/usr/lib/python3.10/threading.py", line 973, in _bootstrap
    self._bootstrap_inner()
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.10/socketserver.py", line 683, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python3.10/socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python3.10/socketserver.py", line 747, in __init__
    self.handle()
  File "/usr/local/lib/python3.10/dist-packages/werkzeug/serving.py", line 342, in handle
    BaseHTTPRequestHandler.handle(self)
  File "/usr/lib/python3.10/http/server.py", line 433, in handle
    self.handle_one_request()
  File "/usr/local/lib/python3.10/dist-packages/werkzeug/serving.py", line 374, in handle_one_request
    self.run_wsgi()
  File "/usr/local/lib/python3.10/dist-packages/werkzeug/serving.py", line 319, in run_wsgi
    execute(self.server.app)
  File "/usr/local/lib/python3.10/dist-packages/werkzeug/serving.py", line 308, in execute
    application_iter = app(environ, start_response)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/http.py", line 2189, in __call__
    response = request._serve_db()
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/http.py", line 1765, in _serve_db
    return service_model.retrying(self._serve_ir_http, self.env)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/service/model.py", line 133, in retrying
    result = func()
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/http.py", line 1792, in _serve_ir_http
    response = self.dispatcher.dispatch(rule.endpoint, args)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/http.py", line 1996, in dispatch
    result = self.request.registry['ir.http']._dispatch(endpoint)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/addons/base/models/ir_http.py", line 222, in _dispatch
    result = endpoint(**request.params)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/http.py", line 722, in route_wrapper
    result = endpoint(self, *args, **params_ok)
  File "/home/jasmins/workspace/odoo_17/odoo/addons/web/controllers/dataset.py", line 28, in call_button
    action = self._call_kw(model, method, args, kwargs)
  File "/home/jasmins/workspace/odoo_17/odoo/addons/web/controllers/dataset.py", line 20, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/api.py", line 468, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/api.py", line 453, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "<decorator-gen-71>", line 2, in button_immediate_install
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/addons/base/models/ir_module.py", line 75, in check_and_log
    return method(self, *args, **kwargs)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/addons/base/models/ir_module.py", line 466, in button_immediate_install
    return self._button_immediate_function(self.env.registry[self._name].button_install)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/addons/base/models/ir_module.py", line 590, in _button_immediate_function
    registry = modules.registry.Registry.new(self._cr.dbname, update_module=True)
  File "<decorator-gen-16>", line 2, in new
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/tools/func.py", line 87, in locked
    return func(inst, *args, **kwargs)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/modules/registry.py", line 113, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/modules/loading.py", line 480, in load_modules
    processed_modules += load_marked_modules(env, graph,
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/modules/loading.py", line 364, in load_marked_modules
    loaded, processed = load_module_graph(
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/modules/loading.py", line 222, in load_module_graph
    module._check()
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/addons/base/models/ir_module.py", line 842, in _check
    if not module.description_html:
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/fields.py", line 1207, in __get__
    self.compute_value(recs)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/fields.py", line 1389, in compute_value
    records._compute_field_value(self)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/models.py", line 4867, in _compute_field_value
    fields.determine(field.compute, self)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/fields.py", line 102, in determine
    return needle(*args)
  File "/home/jasmins/workspace/odoo_17/odoo/odoo/addons/base/models/ir_module.py", line 178, in _get_desc
    warnings.warn(
 
```

This type of warnings are coming

Reply:
Updated, Thanks